python/GafferArnoldTest/IECoreArnoldPreviewTest/RendererTest.py

```diff
@@ -1435,6 +1435,26 @@ def testStepSizeAttributeDefersToProceduralParameter( self ) :
 			self.assertTrue( arnold.AiNodeIs( shape, "volume" ) )
 			self.assertEqual( arnold.AiNodeGetFlt( shape, "step_size" ), 0.25 )
 
+	def testDeclaringCustomOptions( self ) :
+
+		r = GafferScene.Private.IECoreScenePreview.Renderer.create(
+			"IECoreArnold::Renderer",
+			GafferScene.Private.IECoreScenePreview.Renderer.RenderType.SceneDescription,
+			self.temporaryDirectory() + "/test.ass"
+		)
+
+		r.option( "ai:declare:myCustomOption", IECore.StringData( "myCustomOptionValue" ) )
+
+		r.render()
+		del r
+
+		with IECoreArnold.UniverseBlock() :
```

Comment:
Should be `with IECoreArnold.UniverseBlock( writable = True ) :`, since loading the ass file edits the universe.